### DISCUSSION TITLE
Get Queue Request

### DISCUSSION THREAD

**Discussion by TUBEOF** [spotify-web-api-java/spotify-web-api-java#309] [Ideas]
Hi. is there any plan / ETA to implement the Get Queue Request?
https://developer.spotify.com/documentation/web-api/reference/#/operations/get-queue

**Comment by dargmuesli** [comment]
No, but an implementation as a community contribution is possible!
As now mentioned in the README first, it would be great if you could [upvote this ticket](https://community.spotify.com/t5/Spotify-for-Developers/Revive-the-Object-Model-Specification/td-p/5368580), so that there is a reference again against which we can implement features.

Edit: I just saw that you already upvoted the ticket. Thanks!
